brd-ios/breadwallet/src/ViewControllers/KYC/KYCForgotPassword/Cells/KYCForgotPasswordView.swift

```diff
@@ -85,5 +85,10 @@ class KYCForgotPasswordView: BaseView, GenericSettable {
     func changeFieldStyle(isViable: Bool, isFieldEmpty: Bool) {
         emailField.setEmptyErrorMessage(isFieldEmpty: isFieldEmpty)
         emailField.setCheckMark(isVisible: isViable)
+        
+        let isWrongFormat = !isViable && !isFieldEmpty
+        if isWrongFormat {
+            emailField.setDescriptionMessage(isWrongFormat: isWrongFormat)
```

Comment:
can it be just emailField.setDescriptionMessage(isWrongFormat: isWrongFormat) without the if statement? so, when it is false it does not set the message but set if it is true

Reply:
no, it can't. I use (if isWrongFormat) because it is the same error label used when the field is empty and if I put it like you said it will not show the empty field error message, it will be hidden.